core/client/app/templates/setup/two.hbs

```diff
@@ -46,8 +45,5 @@
         {{/gh-form-group}}
     </form>
 
-    <button type="submit" class="btn btn-green btn-lg btn-block" {{action "setup"}} {{submitting}}>Last step: Invite your team <i class="icon-chevron"></i></button>
-    {{#if showError}}
-    <p class="main-error">{{invalidMessage}}</p>
-    {{/if}}
-</section>
+<button type="submit" class="btn btn-green btn-lg btn-block" {{action "setup"}} {{submitting}}>Last step: Invite your team <i class="icon-chevron"></i></button>
```

Comment:
I know it was here previously but what is `{{submitting}}` supposed to be doing inside the button element?